src/resolvers/Mutation/joinPublicOrganization.ts

```diff
@@ -49,15 +48,6 @@ export const joinPublicOrganization: MutationResolvers["joinPublicOrganization"]
       );
     }
 
-    // Checks whether organization is public.
-    if (organization.isPublic === false) {
```

Comment:
shouldn't this check be replaced with `organization.userRegistrationRequired === true` instead of being removed?